packages/utils/src/lib/utils.ts

```diff
@@ -0,0 +1,35 @@
+import isMobile from "is-mobile";
+import { utils, transactions as nearTransactions } from "near-api-js";
+import { Action } from "@near-wallet-selector/core";
+import { createAction } from "@near-wallet-selector/wallet-utils";
+
+interface CreateTransactionParams {
+  accountId: string;
+  publicKey: string;
+  receiverId: string;
+  nonce: number;
+  actions: Array<Action>;
+  hash: string;
+}
+
+const createTransaction = ({
```

Comment:
I'm not sure this method is terribly useful in practice. I think it would only make sense if it resolved the `nonce` and `hash` but that's what `signTransactions` does.

Maybe out of scope for this task, but we could investigate if NEAR Walle could just use `signTransactions` instead. I'd just suggest reverting the changes in `near-wallet` to just use the `createTransaction` directly from `near-api-js`